src/Meilisearch/Autocomplete/Configuration/Configuration.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusMeilisearchPlugin\Meilisearch\Autocomplete\Configuration;
+
+final class Configuration
+{
+    /** @var list<Source> */
+    public array $sources = [];
+
+    public function __construct(
+        public readonly string $host,
+        public readonly string $searchKey,
+
+        /**
+         * This is the javascript selector for the HTML element that will contain the autocomplete
+         */
+        public readonly string $container,
+
+        /**
+         * This is the placeholder text that will be displayed in the input field
+         */
```

Comment:
Also here, both of the comments are redundant in my opinion :dancer:

Reply:
Aaaaaah, not quite. The name of the class is quite generic, so you wouldn't be completely sure unless you traversed some files. But I will make them one liners just for you <3